lib/types.js

```diff
@@ -1077,7 +1077,9 @@ class StringType extends PrimitiveType {
   }
 
   random () {
-    return RANDOM.nextString(RANDOM.nextInt(32));
+    return RANDOM.nextString(
+      Math.floor(-Math.log(RANDOM.nextFloat()) * 16) + 1
+    );
```

Comment:
I touched on this in the PR description above, but this is solely for benchmarking purposes--when rewriting and optimizing the string encoding/decoding functions, I wanted to make sure I exercised both the "short string" and "long string" code paths. This is an exponential distribution, which means that shorter strings are more likely but longer ones are possible.

I can revert this, although in the long term it might be better for `Type#random` to be moved into the testing code, since I'm not sure what purpose it serves to users of the library.

Reply:
Thanks for the background. This is fine. Agreed that this method would be best moved out of the core types, but this is best done separately.